### ISSUE TITLE
Figure 1 discussions

### ISSUE THREAD

**Issue by gwaygenomics** [broadinstitute/grit-benchmark#23]
In #18 I added the first draft of figure 1, lets use this issue to discuss.

![figure1](https://user-images.githubusercontent.com/7340421/108393967-2f987400-71e2-11eb-859d-65034a81c957.png)

**Comment by AnneCarpenter** [comment]
For part a, are replicates replicates or are they different guides? Or it depends on whether you're calculating the Grit of a guide vs the Grit of a gene?

Is it intentional that the lines connect the target to each control (vs connecting each replicate to each control)?

I agree with the concern mentioned today about using median vs mean to insulate a bit against cases where  a single guide is 'bad' aka ineffective. OTOH, one would want a case of 3 great guides to yield a higher score than 2 great and one terrible guide, so 🤷

**Comment by gwaygenomics** [comment]
> For part a, are replicates replicates or are they different guides? Or it depends on whether you're calculating the Grit of a guide vs the Grit of a gene?

Yes, good point. I use "replicate" here for convenience in understanding. It really is "group". "replicate" is _a kind of_ "group". So, you can calculate grit w.r.t. replicates (aka same guide different well) or w.r.t. to the gene group (aka all wells of different guides targeting the same gene).

The fact that the software handles this seamlessly is another benefit :)

> Is it intentional that the lines connect the target to each control (vs connecting each replicate to each control)?

Yep! Every target gets a grit score. In calculating the target's grit score, we never perform calculations directly between the replicate profiles and the controls. It truly is the target's reproducibility within the group with respect to controls.

> I agree with the concern mentioned today about using median vs mean to insulate a bit against cases where a single guide is 'bad' aka ineffective. OTOH, one would want a case of 3 great guides to yield a higher score than 2 great and one terrible guide, so 🤷

Yeah... exactly. Also, at least in the Cell Health data, and empirically, it makes [very little difference](https://raw.githubusercontent.com/broadinstitute/grit-benchmark/main/2.compare-metrics/cell-health/figures/cell_health_grit_metric_summary_comparison.png). Spearman correlations ~.98 ([cell 7](https://github.com/broadinstitute/grit-benchmark/blob/main/2.compare-metrics/cell-health/8.compare-replicate-summary-metric.ipynb))
